packages/components/ng-ovh-responsive-popover/src/class/directive.js

```diff
@@ -1,58 +0,0 @@
-/**
- *  @ngdoc directive
- *
- *  @name responsivePopover.directive:responsivePopoverClass
- *
- *  @description
- *  This directive manage the way the popover is displayed. This uses the configuration setted into
- *  the `responsivePopoverProvider` to detect if the popover needs to be displayed on full screen
- *  or to be displayed normally.
- *  It is automatically setted by `responsivePopover` directive.
- *
- *  **Note:** when ui-bootstrap version >= 2.0 will be used, the only thing to do should be to add
- *  a custom class with the 'popover-class' option of the uibPopover directive. To be tested.
- */
-export default /* @ngInject */ function(matchmedia, responsivePopover) {
-  return {
-    restrict: 'C',
-    link(scope, iElement) {
-      const mobileMediaQuery = responsivePopover.getMobileMediaQuery();
-      let isMobileDetected = false;
-      let mediaQueryHandler = false;
-      let scrollPosition;
-      const resetFixedBody = function(scrollTo) {
-        $('body')
-          .removeClass('responsive-popover-fixed-body')
```

Comment:
Can find this className into following file:
- packages/components/ng-ovh-responsive-popover/src/index.less